cucumber-core/src/main/java/io/cucumber/core/options/Constants.java

```diff
@@ -130,7 +130,8 @@ public final class Constants {
     /**
      * Property name formerly used to pass command line options to Cucumber:
      * {@value} This property is no longer read by Cucumber. Please use any of
-     * the
+     * the individual properties instead.
+     * See the java doc on io.cucumber.core.options.Constants for details.
```

Comment:
Pretty good job on reconstructing the original sentence. But this is the Javadoc of io.cucumber.core.options.Constants so telling people to look there for details isn't quite right.